service/user/internal/src/main/resources/liquibase/common-properties.xml

```diff
@@ -17,5 +17,5 @@
 
     <property name="now" value="sysdate" dbms="oracle"/>
     <property name="now" value="now()" dbms="postgresql"/>
-    <property name="now" value="current_timestamp()" dbms="mysql,mariadb,h2"/>
+    <property name="now" value="current_timestamp(3)" dbms="mysql,mariadb,h2"/>
```

Comment:
Pleae change the header to: 
<!--
    Copyright (c) 2017, 2020 Eurotech and/or its affiliates and others
    All rights reserved. This program and the accompanying materials
    are made available under the terms of the Eclipse Public License v1.0
    which accompanies this distribution, and is available at
    http://www.eclipse.org/legal/epl-v10.html
    Contributors:
        Eurotech - initial API and implementation
 -->